Sources/Scyther/Components/Cells/SliderCell.swift

```diff
@@ -32,23 +32,29 @@ class SliderCell: UITableViewCell {
     }
     
     private func setupConstraints() {
-        textLabel?.snp.remakeConstraints({ (make) in
-            make.top.equalToSuperview().inset(16)
-            make.left.equalToSuperview().inset(16)
-            make.right.equalToSuperview().inset(16)
-        })
+        guard let textLabel = textLabel else { return }
```

Comment:
Any reason for unwrapping here? Preference is to leave the optional in place.

Reply:
the classic way to set layoutconstraints unfortunately does not work with optionals, but i can do it like this if preferred:

```
        if let textLabel = textLabel {
            NSLayoutConstraint.activate([
                textLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16),
                textLabel.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16),
                textLabel.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -16)
            ])
        }
```

Reply:
but in this case, cause all the constraint refer to the textLabel i think its cleaner to catch this with a precondition.

Reply:
That doesn't sound right. I think the issue is that you're trying to do it via the array invocation `NSLayoutConstraint.activate([])`. You could do

```
textLabel?.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16).isActive = true,
textLabel?.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: 16).isActive = true,
textLabel?.rightAnchor.constraint(equalTo: contentView.rightAnchor, constant: -16).isActive = true
```

I still remain unconvinced of the value that this change brings. At this point the complexity that the changes would introduce to the code seem like they negate any benefits that these changes would bring.

Reply:
Good point - if you prefer it this way i will change that